src/app/src/components/EmbeddedFooter.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import logo from '../styles/images/OAR_PoweredBy_white.svg';
+
+const EmbeddedFooter = () => (
+    <footer className="footerContainerEmbedded results-height-subtract" xs={12}>
+        <img
+            className="footer-image"
+            src={logo}
+            alt="Powered by Open Apparel Registry"
+        />
+    </footer>
+);
+
+export default EmbeddedFooter;
```

Comment:
I thought about integrating this into the existing footer component, but the code for that component is somewhat complex, and it seemed simpler to make a new component.

Reply:
That seems like a good call

Reply:
That seems clearer in terms of readibility, as well